src/Annotator/reducers/general-reducer.js

```diff
@@ -13,6 +13,15 @@ import convertExpandingLineToPolygon from "./convert-expanding-line-to-polygon"
 const getRandomId = () => Math.random().toString().split(".")[1]
 
 export default (state: MainLayoutState, action: Action) => {
+  if (action.type === "ON_CLS_ADDED" && action.cls && action.cls !== "") {
+    const oldRegionClsList = state.regionClsList
+    const newState = {
+      ...state,
+      regionClsList: oldRegionClsList.concat(action.cls),
```

Comment:
That's so clear and understandable :dancers:

Reply:
Thanks a lot @puskuruk